### PR TITLE
Configure Styled components to work with Next

### PR DESCRIPTION
**Pull request by eevajonnapanula**

### CODE REVIEW

File: src/pages/_document.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  }
+
+  render() {
+    return (
+      <Html lang="en">
+        <Head>
+          <link
+            href="https://fonts.googleapis.com/css2?family=Lora:wght@700&family=Roboto:wght@400;500;700&dispaly=swap"
+            rel="stylesheet"
+          />
```
[the remark this answers is not in the document]
Comment (eevajonnapanula) [reply src/pages/_document.jsx:37]: The fonts were working for me, but now that you asked I did some googling, and added this `.storybook/preview-head.html` just to be sure 😊

File: src/pages/_document.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Document, { Head, Html, Main, NextScript } from "next/document";
+import { ServerStyleSheet } from "styled-components";
+
+export default class MyDocument extends Document {
+  static async getInitialProps(ctx) {
+    const sheet = new ServerStyleSheet();
+    const originalRenderPage = ctx.renderPage;
+    try {
```
Comment (laylapku) [inline src/pages/_document.jsx:9]: Does this try block enable applying the styles of styled components to the react app? (just trying to learn more about Next :D)

Comment (eevajonnapanula) [reply src/pages/_document.jsx:9]: If I understood correctly, it collects the styles from the components and injects them to the app, and this has lot to do with server side rendering. I was following these instructions: https://styled-components.com/docs/advanced#nextjs 

So yes, I think it does  :D I'm not that deep into Next yet, so I'm sorry I can't right now give more detailed answer 😅

Comment (laylapku) [reply src/pages/_document.jsx:9]: Of course, I've only heard of it, no real practice 😅. Thanks for sharing!

**Pull request merged**